draft-reddy-cose-jose-pqc-hybrid-hpke.md

```diff
@@ -113,6 +113,40 @@ For readability the algorithm ciphersuites labels are built according to the fol
 
 The HPKE PQ/T hybrid ciphersuites for JOSE and COSE are defined in {{IANA}}. Note that the PQ/T Hybrid KEM in HPKE is not an authenticated KEM. The HPKE Base mode can only be supported with the PQ/T Hybrid KEM.
 
+
+# AKP Key for X-Wing 
+
+This section describes the required parameters for an "AKP" key type, as defined in {{!I-D.ietf-cose-dilithium}}, and its use with the X-Wing algorithms, as defined in {{IANA}}. An example JWK is also provided for illustration.
+
+### Required Parameters
+
+A JSON Web Key (JWK) or COSE_Key with a key type ("kty") for use with the "X-Wing" algorithm includes the following parameters:
+
+- kty (Key Type)  
+  The key type parameter MUST be present and set to "AKP".  
+
+- alg (Algorithm)  
+  The algorithm parameter MUST be present and it MUST represent the X-Wing algorithm, as defined in {{IANA}}.  
+
+- pub (Public Key)  
+  The public key parameter MUST be present and contain the public key material in a format defined by the "AKP" key type.  
+
+- priv (Private Key)  
+  The private key parameter MUST be present and contain the private key material in a format defined by the "AKP" key type.  
```

Comment:
I think priv here, should be `sk` (the seed) used to derive ML-KEM and X25519 private keys, as described in the same section I noted above, for example:

```
GenerateKeyPair() returns the 32 byte secret decapsulation key sk
```

Reply:
"priv" is X-Wing decapsulation key (as defined by draft-connolly-cfrg-xwing-kem-06). That is 32-bytes.

Reply:
Yes, updated text to say it is X-Wing decapsulation key (sk).